index.js

```diff
@@ -94,22 +96,22 @@ export default class SlimSlider{
     this.slider.parentNode.style.direction = this.options.dir;
     this.slides.forEach( (el, k) => {
       el.dataset.item = k;
+      el.style.minWidth = `${this.itemWidth}px`;
```

Comment:
I'm using the `min-width : 100%` in css to make sure all slides takes initially full width